flutter/Graminsta/lib/auth/login.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+
+class LoginPage extends StatefulWidget {
+  @override
+  _LoginPageState createState() => new _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.all(20),
+      child: Column(
+        children: <Widget>[
+          _LoginForm(),
+          MaterialButton(
+            color: Colors.blue,
+            textColor: Colors.white,
+            child: Text('Login'),
+            onPressed: () {},
```

Comment:
I think you can omit this field if you don't have callback yet.

Reply:
It seems that the field onPressed is required for material button.